html/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM komand/python-3-plugin
+FROM komand/python-pypy3-full-plugin
```

Comment:
Why do you need the full SDK image?

Reply:
I'm not 100% I do, but it bombed when it was using the older image. I tried building it with slim and condensed and it failed. I stuck with what was there originally.